app/main.py

```diff
@@ -1,3 +1,58 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: (int, float)) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: (int, float, Distance)) -> Distance:
```

Comment:
Use `|`
`int | float | Distance`

Reply:
Should I use "|" every time I have multiple arguments or when there are more than 2?
Should I use it every time instead of a coma?
I hope I don't need to redo this for all my homeworks)
Should I use it in all parts of the methods/functions?
![image](https://github.com/mate-academy/py-distance-class/assets/74096236/2a5e1558-6e26-41c5-8254-94e466a20e0a)